test/NuGet.Core.Tests/NuGet.Resolver.Test/ResolverTests.cs

```diff
@@ -229,6 +229,46 @@ public void ResolveDependenciesForLargeSet()
         }
 
         [Fact]
+        public void ResolveDependenciesForVeryLargeSet()
```

Comment:
Name the test to explicitly reflect the scenario: VeryLargeSet doesn't say much. Is it:

a. Very deep graph?
b. many packages?
c. both (and then you should split it into two tests)